sycl/include/CL/sycl/ONEAPI/matrix/matrix-amx.hpp

```diff
@@ -62,7 +63,7 @@ ELEMS_PER_DWORD(unsigned short, 2)
 
 } // namespace detail
 
-namespace matrix {
+namespace experimental::matrix {
```

Comment:
Since you are nesting the namespaces in this manner, do you get this error
error: nested namespace definitions only available with -std=c++1z or -std=gnu++1z [-Werror=pedantic]

because of the macro, the code in the header files does not get compiled as part of the compiler build, so do you get a warning when you compile the code test?

Reply:
@dkhaldi, In fact, https://github.com/intel/llvm/blob/sycl/sycl/doc/GetStartedGuide.md#prerequisites said gcc 7.1+ is required and gcc7.1 has full support for c++17 which enables nesting namespaces. So I think the line is ok.